.github/workflows/master.yml

```diff
@@ -90,6 +90,7 @@ jobs:
         if: ${{ matrix.codecov == 'true' }}
         run: bash <(curl -s https://codecov.io/bash)
       - name: Detected Dependency List Change
+        if: ${{ matrix.java == 8 }}
```

Comment:
is this already in master？ we can restore this change

Reply:
I think the `git merge` will auto-resolve this change